clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -118,8 +118,8 @@ public Selector(int maxReceiveSize, long connectionMaxIdleMs, Metrics metrics, T
         this.metricsPerConnection = metricsPerConnection;
     }
 
-    public Selector(long connectionMaxIdleMS, Metrics metrics, Time time, String metricGrpPrefix, Map<String, String> metricTags, ChannelBuilder channelBuilder) {
-        this(NetworkReceive.UNLIMITED, connectionMaxIdleMS, metrics, time, metricGrpPrefix, metricTags, true, channelBuilder);
+    public Selector(long connectionMaxIdleMS, Metrics metrics, Time time, String metricGrpPrefix, ChannelBuilder channelBuilder) {
+        this(NetworkReceive.UNLIMITED, connectionMaxIdleMS, metrics, time, metricGrpPrefix, metrics.config().tags(), true, channelBuilder);
```

Comment:
@junrao Thanks for your detailed comment. I have verified that the selectors created in `ReplicaFetcherThread` and `ControllerChannelManager` are still correct w/o including default tags -- they use metrics with empty default tag.

In the selector constructor in this code snippet, the tags that were previously explicitly specified (e.g. in KafkaConsumer and KafkaProducer) are now passed via the default tag in the metrics name. In other words, I simplified the this Selector constructor signature to exclude `Map<String, String> metricTags`, and passed `metrics.config().tags()` to the other Selector constructor. Is this OK?